UI/cypress/e2e/shortlist.feature

```diff
@@ -80,7 +80,7 @@ Feature: Tuition Partner shortlist
         And they choose to view their shortlist from the results page
         When they choose to view the 'Action Tutoring' details from the shortlist
         Then the back link's text is 'Back to shortlist'
-        And the heading caption is 'Tuition partner for Stockport'
+        And the heading caption is 'Shortlisted tuition partner for Stockport'
```

Comment:
Do we not need some new cypress tests for this PR?

Reply:
I will look into this👍